components/TutorialList.js

```diff
@@ -0,0 +1,54 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+export default function TutorialList({ tutorials }) {
+  return (
+    <ListContainer>
+      {tutorials.map((tutorial) => {
+        return (
+          <ListItem key={tutorial.id}>
+            <ListImage>
+              <Image
+                src={tutorial.cover}
+                alt={tutorial.name}
+                layout="fill"
+                objectFit="cover"
+              ></Image>
+            </ListImage>
+            <ListName>{tutorial.name}</ListName>
+          </ListItem>
+        );
+      })}
+    </ListContainer>
+  );
+}
+
+const ListContainer = styled.ul`
+  padding: 1em;
+  list-style: none;
+  background-color: var(--background);
+`;
+
+const ListItem = styled.li`
+  background-color: var(--primary-50);
+  display: flex;
+  flex-wrap: nowrap;
```

Comment:
this is the default so you can remove it

Reply:
:white_check_mark:

Reply:
Changed to grid. ✅